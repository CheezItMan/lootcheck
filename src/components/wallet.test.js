import React from 'react';
import { shallow } from 'enzyme';
import Wallet from './wallet';

describe('Wallet', () => {
    let wallet;
    const props = {balance: 20,}
    beforeEach(() => {
      wallet = shallow(<Wallet  {...props} />);
    });


    test('it renders properly', () => {
      expect(wallet).toMatchSnapshot();
    });

    test('it displays the balance from props', () => {
      expect(wallet.find('.balance').text()).toEqual("Wallet balance: 20");
    });
});
