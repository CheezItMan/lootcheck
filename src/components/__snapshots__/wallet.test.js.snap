// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wallet it renders properly 1`] = `undefined`;
